integration-test/trino-it/docker-compose.yaml

```diff
@@ -0,0 +1,103 @@
+#
+# Copyright 2023 Datastrato Pvt Ltd.
+# This software is licensed under the Apache License version 2.
+#
+version: '3.0'
```

Comment:
Could we use the test containers of Docker compose?
https://codeal.medium.com/how-to-run-docker-compose-with-testcontainers-7d1ba73afeeb

Reply:
Sometimes, debugging requires manually starting containers. Using scripts can maintain consistency between automated and manual environment setups, and it allows for easy modification of the testing environment. I feel that Testcontainers focuses more on setting up containers programmatically, which differs from our scenario.

Reply:
I'm a little concerned about the stability of the CI. We can also reuse the class `DockerComposeContainer `.